resources/web/wwi/Parser.js

```diff
@@ -924,7 +916,10 @@ export default class Parser {
     }
 
     const ccw = getNodeAttribute(node, 'ccw', 'true').toLowerCase() === 'true';
-    const ifs = new WbIndexedFaceSet(id, coordIndex, normalIndex, texCoordIndex, coordArray, texCoordArray, normalArray, ccw);
+    const normalPerVertex = getNodeAttribute(node, 'normalPerVertex', 'true').toLowerCase() === 'true';
+    const creaseAngle = parseFloat(getNodeAttribute(node, 'creaseAngle', '2'));
```

Comment:
In Webots, the default value for the creaseAngle is 0, not 2.

Reply:
Oops I forgot to change it back after testing